wordpress-mobile/WordPress-Android
# [Blogging Prompts] Create Prompts List Activity placeholder and navigation

Part of #17125

Create a placeholder `Activity` using Compose for the Prompts Lists, create the navigation helpers for opening it, and start this `Activity` in the appropriate callback, passing it to the Blogging Prompts card builder.

There are a couple of TODOs in the PR that should be dealt with in future PRs also targeting issue #17125, implementing the content of this screen. They are:
- Checking if it makes sense to bring in the `androidx.activity:activity-compose` lib
- Create a reusable `TopAppBar` Composable matching the XML app bar specs used in other screens
  - It looks like the one I currently created has some slight differences in terms of color and text letter spacing, see the screenshot below

<details><summary>Screenshot of the topbar differences</summary>

![samescreen_topappbar](https://user-images.githubusercontent.com/5091503/208527249-1325b987-7d4c-4add-8718-6ea832db54bb.png)
</details>

<details><summary>Video Demo</summary>

https://user-images.githubusercontent.com/5091503/208527775-ad99f4d9-a969-42c8-a471-cd84059b26b6.mp4
</details>


To test:
1. Do the steps to enable the Prompts List feature flag found in the test section of #17675
2. Sign in (if not signed in already)
3. Go to the home dashboard (My Site -> Home)
4. **Verify** the Prompts card is shown
5. Tap the 3-dot menu in the Prompts card
6. Tap the "View more prompts" option
7. **Verify** the Prompts List placeholder activity is shown

## Regression Notes
1. Potential unintended areas of impact
N/A

3. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

4. What automated tests I added (or what prevented me from doing so)
- Unit tests checking for callback being propagated
- Updated existing unit tests

PR submission checklist:
